fuzz/build.info

```diff
@@ -89,7 +89,7 @@ IF[{- !$disabled{tests} -}]
 
   SOURCE[asn1-test]=asn1.c test-corpus.c
   INCLUDE[asn1-test]=../include
-  DEPEND[asn1-test]=../libcrypto ../libssl
+  DEPEND[asn1-test]=../providers/libnonfips.a ../libcrypto ../libssl
```

Comment:
Change `../libcrypto ../libssl` to `../libcrypto.a ../libssl.a` to have `asn1-test` link with the static libraries instead of the shared libraries

Reply:
Mega-thanks!  Why don't the shared libraries export them?

Reply:
> Mega-thanks! Why don't the shared libraries export them?

You have placed the declarations in a header file in `include/crypto`, those are libcrypto private declarations.  If you want them exported, place the declarations in a header file in `include/internal`, that'll be picked up automatically.

It depends on what your intent is with the `RAND_swap` functions.

Reply:
Of course, thanks, that public/private explanation makes sense.